bin/next-start

```diff
@@ -10,11 +10,13 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 const argv = parseArgs(process.argv.slice(2), {
   alias: {
     h: 'help',
+    H: 'hostname',
     p: 'port'
   },
   boolean: ['h'],
   default: {
-    p: 3000
+    p: 3000,
+    hostname: 'localhost'
```

Comment:
If there's no default then the second argument to `listen()` will be `undefined`. I'm not sure what that would do.